ci/start-console.sh

```diff
@@ -21,8 +31,8 @@ if oc_available_loggedin; then
 fi
 
 # Default to localhost if no route found
-INVENTORY_SERVER_HOST=${INVENTORY_SERVER_HOST:-"https://localhost:30444"}
-SERVICES_API_SERVER_HOST=${SERVICES_API_SERVER_HOST:-"https://localhost:30446"}
+INVENTORY_SERVER_HOST=${INVENTORY_SERVER_HOST:-"$BASE_HOST_URL:30444"}
+SERVICES_API_SERVER_HOST=${SERVICES_API_SERVER_HOST:-"$BASE_HOST_URL:30446"}
```

Comment:
Here the urls were set by default before the fix to `https://localhost` and not to `http://localhost`,  so you may need two variables or just set the HOST part..

Reply:
changed to https thanks!

Reply:
This is not working now, since the console doesn't find the plugin. It is searching for the plugin on `https://localhost` and the plugin is running using `http://localhost`.
The original line was: 
https://github.com/kubev2v/forklift-console-plugin/blob/cc62d29ef2e11deae2cd3ca6ae6857b69ffa8222/ci/start-console.sh#L11

You need to use different schema for each:
for PLUGIN_URL - http
for BASE_HOST_URL - https